colin-api/src/colin_api/models/reset.py

```diff
@@ -86,6 +86,22 @@ def get_filings_for_reset(self):
             current_app.logger.error('error getting filing/event info for reset for: {}'.format(self.as_dict()))
             raise err
 
+    @classmethod
+    def get_incorporations_by_event(cls, cursor, event_ids: list):
```

Comment:
Move this to after the protected methods pls (for better readability).

Reply:
this actually should be a protected method, I have moved it to the last protected method